tests/roxctl/bats-tests/local/roxctl-connectivity-map-release.bats

```diff
@@ -227,6 +222,180 @@ default/frontend[Deployment] => default/backend[Deployment] : TCP 9090'
   assert_output --partial 'default/frontend[Deployment] => default/backend[Deployment] : TCP 9090'
 }
 
+@test "roxctl-release connectivity-map generates empty connlist netpols blocks ingress connections from Routes" {
+  assert_file_exist "${test_data}/np-guard/security-frontend-demo/asset-cache-deployment.yaml"
+  assert_file_exist "${test_data}/np-guard/security-frontend-demo/asset-cache-route.yaml"
+  assert_file_exist "${test_data}/np-guard/security-frontend-demo/frontend-netpols.yaml"
+  assert_file_exist "${test_data}/np-guard/security-frontend-demo/webapp-deployment.yaml"
+  assert_file_exist "${test_data}/np-guard/security-frontend-demo/webapp-route.yaml"
+  run roxctl-release connectivity-map "${test_data}/np-guard/security-frontend-demo"
+  assert_success
+  # netpols deny connections between the existing deployments; and blocks ingress from external ips or ingress-controller
+  # the output contains only WARN and INFO messages
+  echo "$output" > "$ofile"
+  assert_file_exist "$ofile"
+  assert_output --partial 'Route objects/s: frontend/asset-cache specified workload frontend/asset-cache[Deployment] as a backend, but network policies are blocking ingress connections from an arbitrary in-cluster source to this workload.'
```

Comment:
Let's change this warning in our repo and then here , as follows: `Route resource  frontend/asset-cach` instead of `Route objects/s: frontend/asset-cache`